rs-commerce/src/views/user-profile/UserProfile.tsx

```diff
@@ -0,0 +1,156 @@
+import ProfileComponent from 'components/profile-component/profileComponent';
+import { useAppSelector } from 'hooks/typed-react-redux-hooks';
+import { apiAuthSelector } from 'redux/selectors';
+import { InputProfileType, ProfileComponentType } from 'components/profile-component/types';
+import CustomLink from 'components/custom-link/custom-link';
+import { ROUTE_PATH } from 'constants/constants';
+import classes from './UserProfile.module.css';
+import UserProfileHeader from './user-profile-header/UserProfileHeader';
+
+export default function UserProfile(): JSX.Element {
+  const { userData } = useAppSelector(apiAuthSelector);
```

Comment:
check is userData exist in store and if not use request to get current user data and set it in store. now fields may be empty